DOCS-3149: Update Pool Creation Instructions for disks without unique… (#1320)
truenas/documentation: content/SCALE/Storage/Pools/_index.md

Next, configure the virtual devices (vdevs) that make up the pool.

{{< hint info >}}

> If the disks being used have non-unique serial numbers, they will not populate the *Available Disks* section until *Show disk with non-unique serial numbers* is checked.

Reviewer: reword to make active voice present tense, remove the word will
```suggestion
If the disks used have non-unique serial numbers, they do not populate the **Available Disks** section until the **Show disk with non-unique serial numbers** checkbox is selected.
```
